python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+                        canvas.delete(line3)
+                        canvas.delete(oval2)
+                        canvas.delete(text2)
+                        canvas.delete(distancenew)
+                        oval4=canvas.create_oval(a1,b1,a2,b2,fill="orange")
+                        text4=canvas.create_text(m,n,text=f"{charge2val}")
+                        line3=canvas.create_line(x_axis,y_axis,a1,n,fill="black",width="2")
+                        x11=m
+                        y11=n
+                        f,g=compute.mp(x_axis,y_axis,a1,n)
+                        distnew=int((compute.liner(x_axis,y_axis,a1,n))//60)
+                        distancenew=canvas.create_text(f,g,text=f"{distnew}")
+        bamchik=i.get()
+        bumchik=mango.get()
+        z=compute.getter(bamchik)        
+        force=compute.forcecalc(charge1val,charge2val,z,distnew,bamchik)
+        M.set(str(force))       
+    canvas.bind("<Button-1>",dad)                        
+    root1.mainloop()
+#second window ka mainloop hai confuse mat hona  
+############################################################################################################################################################
+#FIRST WINDOW BY AWASTHIJI 
+root=Tk()
+root.title("COULOMB'S LAW")
+root.minsize(550,350)
+root.maxsize(550,350)
+label1=Label(root,text="Charge1",font="TIMESNEWROMAN 15 bold",bg="grey",fg="black",bd=2,relief="solid")
+label1.place(x=100,y=20)
+label2=Label(root,text="Charge2",font="TIMESNEWROMAN 15 bold",bg="grey",fg="black",bd=2,relief="solid")
+label2.place(x=100,y=60)
+Charge1=IntVar()
+Charge1.set(1)
+entry1=Entry(root,textvariable=Charge1,bd=3,relief="sunken")
+entry1.place(x=190,y=26)
+Charge2=IntVar()
+Charge2.set(1)
+entry2=Entry(root,textvariable=Charge2,bd=3,relief="sunken")
+entry2.place(x=190,y=66)
+box=IntVar()
+label3=Label(root,text="MEDIUM :",font="TIMESNEWROMAN 15 bold",bg="grey",fg="black",bd=2,relief="solid")
+label3.place(x=100,y=100)
+box1=Radiobutton(root,text="Air",value=1,variable=box,bd=2,relief="sunken")
+box1.place(x=210,y=100)
+box4=Radiobutton(root,text="Vaccum",value=2,variable=box,bd=2,relief="sunken")
+box4.place(x=280,y=100)
+box5=Radiobutton(root,text="Free Space",value=3,variable=box,bd=2,relief="sunken")
+box5.place(x=380,y=100)
+label4=Label(root,text="UNIT :",font="TIMESNEWROMAN 16 bold",bg="grey",fg="black",bd=2,relief="solid")
+label4.place(x=100,y=150)
+i=IntVar()
+r1=Radiobutton(root,text="pm",value=123,variable=i,bd=2,relief="sunken")
+r2=Radiobutton(root,text="cm",value=234,variable=i,bd=2,relief="sunken")
+r3=Radiobutton(root,text="mm",value=345,variable=i,bd=2,relief="sunken")
+r4=Radiobutton(root,text="nm",value=456,variable=i,bd=2,relief="sunken")
+r1.place(x=200,y=150)
+r2.place(x=260,y=150)
+r3.place(x=320,y=150)
+r4.place(x=380,y=150)
+mango=IntVar()
+label5=Label(root,text="CHARGE'S UNIT:",font="TIMESNEWROMAN 12 bold",bg="grey",fg="black",bd=2,relief="solid").place(x=50,y=200)
```

Comment:
Codacy found an issue: ['Label' may be undefined, or defined from star imports: tkinter (F405)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)